index.bs

```diff
@@ -501,31 +501,31 @@ dictionary StreamPipeOptions {
 Instances of {{ReadableStream}} are created with the internal slots described in the following
 table:
 
-<table>
+<table dfn-for="ReadableStream">
  <thead>
   <tr>
    <th>Internal Slot
    <th>Description (<em>non-normative</em>)
  <tbody>
   <tr>
-   <td>\[[disturbed]]
+   <td><dfn id="rs-slot-disturbed" lt="[[disturbed]]">\[[disturbed]]</dfn>
```

Comment:
We should probably omit the `id=""`s. I regret manually curating them for abstract operations, and I don't think we should set ourselves up for that maintenance burden going forward.